documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -439,3 +439,82 @@ characteristicDefinitionsUrl.ifPresent( url -> {
 } );
 ----
 
+[[mapping-aas]]
+== Mapping Aspect Models to Asset Administration Shell (AAS) Submodel Templates
+The Asset Administration Shell (AAS) and its https://www.plattform-i40.de/IP/Redaktion/EN/Standardartikel/specification-administrationshell.html[information model] is a widely recognized standard developed by the https://industrialdigitaltwin.org[Industrial Digital Twin Association (IDTA)] to express and handle Digital Twins. Central element of the AAS is
+the concept of Submodels, which describe certain aspects of a Digital Twin.
+
+The BAMM Aspect Meta Model allows to specify aspects of a digital twin and its semantics.
+The AAS Generator module provides mapping implementations to derive AAS Submodels from BAMM Aspect models
+and by that allows to on the one hand integrate BAMM models in AAS environments and on the other hand allow
+AAS Submodels to be described with rich semantics, as it is possible with BAMM.
+
+[[details-mapping-aas]]
+== Details to the Mapping Concept
```

Comment:
This should be one indentation level below, i.e. "===". Also, "Details _of_ the Mapping Concept"?